src/components/MovieDetails/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class MovieDetails extends React.Component {
+    renderDate(stringDate) {
+        return (
+            1900 + (new Date(stringDate)).getYear()
+        )
+    }
+    render() {
+        const { movie } = this.props;
+        return (
+            <div className="MovieDetails">
+                <img src= { movie.poster_path } className="MovieDetails__Image"/>
+                <div className="MovieDetails_desc">
```

Comment:
It is better to follow most common naming convention for BEM and use words in lowercase separated by hyphen and underscore.

[https://en.bem.info/methodology/naming-convention/](https://en.bem.info/methodology/naming-convention/)

Reply:
fixed, but you have to agree that this is quite opinionated - even in the doc there are several naming conventions mentioned=)